mumbleBot.py

```diff
@@ -184,6 +184,7 @@ def message_received(self, text):
                     self.volume = float(float(parameter) / 100)
                     self.send_msg_channel(var.config.get('strings', 'change_volume') % (
                         int(self.volume * 100), self.mumble.users[text.actor]['name']))
+                    var.config.set('bot', 'volume', str(self.volume))
```

Comment:
Not sure about this one. You can have the "junk people" who set the bot volume at 100%.

Reply:
You can still !v 100 though, this doesn't really enable it any easier. Personally I've not had a problem on my server and having to set the bot volume again each restart even if restarts are far apart seems to me more annoying long term.